client/src/containers/PopUp/PopUpEdit.js

```diff
@@ -0,0 +1,51 @@
+import React, {  useState } from "react";
+import editHours from "../../services/axios_sev/editHours";
+import { useForm } from "react-hook-form";
+import { Link } from "react-router-dom";
+import {connect} from 'react-redux'
+
+function PopUpEdit({ userToken, userId, monthData }) {
+  const { register, errors, handleSubmit } = useForm();
+  const [message, setMessage] = useState("");
+
+  /**
+   * @function updateData - send information to the server to  udate the data from the db
+   * @param {*} value  - values that comes from the input
+   * @param {*} e -event
+   * @returns {obj} - returns a message that says if the request was sucessful or not
```

Comment:
It doesn't actually have a return value. You could say it `@returns void`, or return the result of the `editHours` function. Which in that case you would want to say it `@returns Promise<void>` :)